bindata/network/multus/multus.yaml

```diff
@@ -521,50 +522,68 @@ spec:
             name: {{.CniSysctlAllowlist}}
             defaultMode: 0744
 {{if .RenderIpReconciler}}
+
 ---
-apiVersion: batch/v1
-kind: CronJob
+apiVersion: apps/v1
+kind: DaemonSet
 metadata:
-  name: ip-reconciler
+  name: whereabouts-reconciler
   namespace: openshift-multus
   labels:
     tier: node
-    app: whereabouts
+    app: whereabouts-reconciler
 spec:
-  # reconcile loop every 15 minutes, starting at the top of the hour
-  schedule: "*/15 * * * *"
-  concurrencyPolicy: Replace
-  successfulJobsHistoryLimit: 0
-  jobTemplate:
+  selector:
+    matchLabels:
+      name: whereabouts-reconciler
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        tier: node
```

Comment:
Echoing @andreaskaris comment regarding label.

Reply:
So removing the label is still outstanding, because it's still here on the latest commit :-)